client/js/components/statement/publicStatementModal/StatementModal.vue

```diff
@@ -1283,6 +1283,16 @@ export default {
 
       return makeFormPost(this.formData, route)
         .then(response => {
+          if (response.status === 429) {
+            dplan.notify.notify('error', Translator.trans('error.statement.not.saved.throttle'))
+
+            return false;
+          }
+          if (response.status !== 200) {
```

Comment:
are we shure that it can't be any other of the 200s?

Reply:
what do you mean by "it"?

Reply:
can the responstatus also be something like 204

Reply:
In case of Route `DemosPlan_statement_public_participation_new_ajax`? I do not think so. Only 200 should be a successful request

Reply:
ok. Than it should be fine. And if we want to be a b bit softer, we could say `response.status <= 200 && response.status < 400` or something like that.

Reply:
Then it should be `response.status <= 200 && response.status < 300`. Do you feel this would be beneficial?

Reply:
It feels like we may get one or two bugs less in the future. Yes.

Reply:
Thanks for pointing that out. Haven't seen the part below. Than Just leave it like it is. to not make it larger than it is.